src/components/SideBarItems/SideBarItems.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Link, useLocation} from "react-router-dom";
+import styles from './SideBarItemStyles.module.css';
+
+const SideBarItems = () => {
+    const location = useLocation();
+
+    const sideBarItems = [
```

Comment:
We can put this outside and above this component. If it is a constant, we don't need to re-create this every time the component re-renders.